flepimop/gempyor_pkg/src/gempyor/inference_parameter.py

```diff
@@ -65,8 +65,8 @@ def add_modifier(self, pname, ptype, parameter_config, subpops):
                     pname=pname,
                     subpop=",".join(group),
                     pdist=parameter_config["value"].as_random_distribution(),
-                    lb=parameter_config["value"]["a"].get(),
-                    ub=parameter_config["value"]["b"].get(),
+                    lb=float(parameter_config["value"]["a"].get()),
```

Comment:
@emprzy you may want to use `parameter_config["value"]["a"].get(float)` instead of letting .get choose a type and then using the python casting to float (.get is a method from confuse, our config library, that allow you to get a yaml element and you can specify a type. See https://confuse.readthedocs.io/en/latest/usage.html)

Reply:
Understood. I will implement this change